packages/vite-plugin-mcp/src/index.ts

```diff
@@ -23,6 +23,7 @@ export function ViteMcp(options: ViteMcpOptions = {}): Plugin {
   return {
     name: 'vite-plugin-mcp',
     async configureServer(vite) {
+      vite.config.server.port = options.port || vite.config.server.port
```

Comment:
I think the port option should only affect the MCP server, but here it mutates the Vite server as well, I don't feel this is expected.

Reply:
You're absolutely right! my mistake. will fix